### DISCUSSION TITLE
Linking is a bit of a bummer

### DISCUSSION THREAD

**Discussion by zackkrida** [vercel/vercel#3991] [General] [answered]
Previously, `now dev` was an awesome tool to test one-off serverless functions locally. The project linking requirement feels prohibitive here. Totally understand the motivation behind it, just thought I'd bring it up. Love all of zeit's services!

**Comment by styfle** [comment, marked as answer]
Thanks for the feedback!

We are aware of the pain for new projects using `now dev` since Now 17 which requires project linking. We are looking at different ways of solving this issue such as introducing `now link` to create a project without deploying (today you have to deploy with `now` before developing locally which seems backwards).

Related: https://github.com/zeit/now/issues/3767

**Reply by styfle** [reply]
The canary channel has been updated with a new command, `vercel link`.

Additionally, `vercel dev` will ask to link to a project if it is not already.

Try it out now with `npm i -g vercel@canary`, thanks!
